### PR TITLE
Replace hello world handler with real dd/homepage handler

### PR DESCRIPTION
**Pull request by NeilMadden**
### What

Replaces the stub Hello World handler with a (bare-bones) handler for the dd/homepage.

### How to review

Download and `make debug` to build this component, then download and build [the PR against dp-frontend-renderer](https://github.com/ONSdigital/dp-frontend-renderer/pull/9) and then do a GET against `localhost:20030/dd/homepage` to see the rendered result (can do this directly in your browser). NB: you will need to run sixteens to get the logo and stylesheets to display properly.

### Who can review

Anyone but me.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func main() {
 		requestID.Handler(16),
 	).Then(router)
 
-	router.Get("/dd/hello", hello.Handler)
+	router.Get("/dd/homepage", homepage.Handler)
```
Comment (CarlHembrough) [inline main.go:37]: Return a 404 if the route is not matched? Maybe using a default handler to return 404?

File: handlers/homepage/handler.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/dp-frontend-models/model/dd/homepage"
+)
+
+// Handler handles requests to the homepage
+func Handler(w http.ResponseWriter, req *http.Request) {
+	page := homepage.Homepage{}
+
+	body, err := renderer.Render(page, "dd/homepage")
+	if err != nil {
```
Comment (CarlHembrough) [inline handlers/homepage/handler.go:15]: If the renderer returns a 404 we should pass this back through the controller

File: main.go (hunk 1 above)
Comment (NeilMadden) [reply main.go:37]: Yes, will fix.

File: handlers/homepage/handler.go (hunk 2 above)
Comment (NeilMadden) [reply handlers/homepage/handler.go:15]: Yes, will fix.

Comment (NeilMadden) [reply handlers/homepage/handler.go:15]: Actually, in this case we are calling the renderer for what should be a known template, so a 404 is a misconfiguration of the system (we're trying to use a template that doesn't exist). I'll improve the error logging though.

**Review by CarlHembrough — APPROVED** [review]
Run 'go fmt' and update the vendor directory for dp-frontend-models

**Comment by NeilMadden** [comment]
I've merged in Lloyd's changes, which includes vendoring in all dependencies. I've updated the homepage model vendor dependency.

**Pull request merged**